app/repositories/code_chunks.py

```
@@ -0,0 +1,76 @@
+            logging.error(f"Similarity search failed: {e}")
+            return []  # Return empty list on error
+
+
+    async def get_repo_file_chunks(self,  user_id : Union[str, UUID] , repo_id: Union[str, UUID],  file_name:str="readme"):
+        """Return chunks of a specific file"""
+        try:
+            result = await CodeChunks.filter( file_name__icontains=file_name,  user_id=user_id, repo_id=repo_id).order_by("-created_at").values("content")
+
+
+            return result
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)